kore/src/Kore/Syntax/Bytes.hs

```diff
@@ -0,0 +1,95 @@
+{-|
+Copyright   : (c) Runtime Verification, 2019
+License     : NCSA
+-}
+
+module Kore.Syntax.Bytes
```

Comment:
This should be `Kore.Internal.InternalBytes` because it is an internal representation and we usually have the module name match the type name.